client/lib/analytics/ad-tracking/record-hosting-trial-started.ts

```diff
@@ -0,0 +1,18 @@
+import { redditTrackerFreeTrialStarted } from './reddit';
+
+/**
+ * Flow names for free hosting trials. Can be used in third-party trackers to distinguish between different flows,
+ * e.g. whether it's an import of an existing site, or a new site.
+ */
+export enum FlowNames {
+	NewSite = 'new_site',
+	SiteMigration = 'site_migration',
+}
```

Comment:
We try and avoid enums p7H4VZ-3sK-p2.

```suggestion
export type FlowNames = 'new_site' | 'site_migration';
```

Reply:
Thanks! I wasn't aware 👍 I've switched it to a string union type now.